content/Engineering/Networking/databases.md

```diff
@@ -32,11 +32,13 @@ This is Opinionated Guides, so let me be clear, I really, **really** hate SQL. I
 
 ![XKCD Drop Tables](/eng/xkcddrop.webp)
 
-{{< attribution >}}
+##{{< attribution >}}
+##Permanent link to this comic: https://xkcd.com/327/
+##Image URL (for hotlinking/embedding): https://imgs.xkcd.com/comics/exploits_of_a_mom.png
 
 [TODO] need image source
 
-{{< /attribution >}}
+##{{< /attribution >}}
 
```

Comment:
I don't know why you `##`'d these. Without a space they're not even headings ... not that they should be?

Reply:
Oh no, it's just how I was laying it out to organize it, those aren't supposed to be "headings" in the final. It's mostly just a progress report. 

Is it okay if I use the OP Guides .md style for cohesion?